samples/net/coap_client/src/coap-client.c

```diff
@@ -42,14 +42,23 @@ static struct net_mgmt_event_callback cb;
 
 static const char * const test_path[] = { "test", NULL };
 
+volatile u8_t test_complete = 0;
 static void msg_dump(const char *s, u8_t *data, unsigned len)
 {
 	unsigned i;
 
-	printk("%s: ", s);
+	printk(" msg_dump =  %s \n ", s);
 	for (i = 0; i < len; i++)
 		printk("%02x ", data[i]);
-	printk("(%u bytes)\n", len);
+
+	printk("\n");
+
+	for (i = 0; i < len; i++)
```

Comment:
Please add {} even for one liners. The original code did not had these but if you make changes then it is worth adding those braces.